Woof/WoofTests/Model/Preferences/PreferencesHandlerSaveTests.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+
+final class PreferencesHandlerSaveTests: XCTestCase {
+    private var userPreferencesStorage = KeyValueStorage.preferencesStorage
+
+    override func setUp() {
+        super.setUp()
+        userPreferencesStorage.removeAll()
+    }
+
+    override func tearDown() {
+        userPreferencesStorage.removeAll()
+        super.tearDown()
+    }
+
+    func testSaveSelectedRoleExistsInAPI() {
+        // Given
```

Comment:
the name of the test doesn't match the method name that we call, review other tests.